extension/ballastextension/config.go

```diff
@@ -18,11 +18,15 @@ import (
 	"go.opentelemetry.io/collector/config"
 )
 
-// Config has the configuration for the fluentbit extension.
+// Config has the configuration for the ballast extension.
 type Config struct {
 	config.ExtensionSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
 
-	// SizeMiB is the size, in MiB, of the memory ballast
+	// BallastSizeMiB is the size, in MiB, of the memory ballast
 	// to be created for this process.
-	SizeMiB uint32 `mapstructure:"size_mib"`
+	BallastSizeMiB uint32 `mapstructure:"ballast_size_mib"`
```

Comment:
Do we need to rename this? memory_ballast.ballast_size_mib is repetitive. `memory_ballast.size_mib` is clear that it's the blast size.

Reply:
ack